python/kwiver/vital/types/CMakeLists.txt

```
@@ -176,28 +186,30 @@ target_link_libraries(python-vital.types-object_track_set
   PUBLIC ${PYTHON_LIBRARIES}
          vital)
 
-kwiver_create_python_init(vital/types
-  types
-  bounding_box
-  camera
-  camera_intrinsics
-  camera_map
-  color
-  covariance
-  descriptor
-  descriptor_set
-  detected_object_type
-  detected_object
-  detected_object_set
-  eigen
-  feature
-  homography
-  homography_f2f
-  landmark
-  rotation
-  similarity
-  timestamp
-  track
-  track_set
-  object_track_set
-)
+if(NOT SKBUILD)
+  kwiver_create_python_init(vital/types
+    types
+    bounding_box
+    camera
+    camera_intrinsics
+    camera_map
+    class_map
+    color
+    covariance
+    descriptor
+    descriptor_set
+    detected_object
+    detected_object_set
+    eigen
+    feature
+    homography
+    landmark
+    rotation
+    similarity
+    timestamp
+    track
+    track_set
+    object_track_set
+    activity
```

Could a comment be add explaining that these lines are intentionally out of order and why that was done? E.g.:

```suggestion
    # These modules depend on some of the ones above and therefore
    # must be loaded after them.  (In particular, object_track_set
    # depends on track_set, and activity depends on object_track_set.)
    object_track_set
    activity
```